tests/faultydata/variants/duplicated_sampleid/README.md

```diff
@@ -0,0 +1,6 @@
+Data file example_1.vcf.gz was copied from tests/data/variants/example_1, then change sample id S2 to S1
+
+Index file was generated by
+```
```

Comment:
Actually, one other thing... does it make sense to have these files as .md files and format them in markdown?

I'm not sure if we should be formatting the README files (aside from the ones in the root directory) as anything other than .txt ... @jeromekelleher thoughts?